packages/office-ui-fabric-react/src/components/Button/CommandButton/CommandButton.scss

```diff
@@ -9,59 +9,61 @@
 
 $button-commandButtonHeight: 40px;
 
-.ms-Button.ms-Button--command {
-  @include text-align(left);
+:global {
```

Comment:
Nitpick. You are adding a new line after the ":global { " in other files but not this one. I would suggest making it consistent